### PR TITLE
Backport: Change doc file names

### PR DESCRIPTION
**Pull request by chenette**
Merge PR #1390 into 1-0.

### CODE REVIEW

**Comment by chenette** [comment]
Yes, there are several doc PRs that depend on these file-name changes.  This one is part of the big doc reorg (PR #1392), but affects several others. 

The details and history: Shawn asked me last week to do these name changes in a separate PR so it would be easy to cherry-pick for 1.0.  (Because URLs depend on file-name changes, so we wanted to avoid breaking links in case someone has bookmarked a specific page.) But we decided not to backport this until after the 1.0 release. 

In summary, this PR is needed for several others that have substantive doc changes and corrections.

**Comment by dcmiddle** [comment]
Thanks. Just up to @vaporos now.

**Comment by chenette** [comment]
Closing this PR because the backporting might happen in a different way.
